### ISSUE TITLE
Changes to CustomVisuals not getting rebuilt on watch

### ISSUE THREAD

**Issue by jdrake** [Microsoft/PowerBI-visuals#355]
The `/src/Clients/PowerBIVisualsPlayground/obj/CustomVisuals.js` file doesn't seem to get recompiled after a watch trigger:

Running `gulp watch`:
```bash
[16:16:16]   Waiting for changes...
[16:19:22] Starting 'tslint:customVisuals-on-watch'...
[16:19:22] Starting 'build:customVisuals-ts'...
[16:19:22] Starting 'tslint:customVisuals-on-watch'...
[16:19:37] Finished 'tslint:customVisuals-on-watch' after 15 s
[16:19:37] Finished 'tslint:customVisuals-on-watch' after 15 s
[16:19:50] Finished 'build:customVisuals-ts' after 28 s
```

I have to run `gulp build` for changes to get picked up.

Running Node 4.2 on OS X 10.10.

**Comment by ignatvilesov** [comment]
@jdrake Thanks, we added this issue to backlog.

**Comment by ignatvilesov** [comment]
@jdrake This problem is fixed in PR #441.
